src/CustomPostMultipleFormats.hs

```diff
@@ -53,8 +49,8 @@ instance MimeUnrender ANewFormat String where -- This instance implements the de
     Right x -> Right $ ("Decoded from ANewFormat - " ++ (T.unpack $ TL.toStrict x)) -- We just prefix the decoded text to differentiate it to show this was decoded using ANewFormat decoding logic.
     Left _ -> Left "Decoding error"
 
-handlerName :: String -> Handler String 
-handlerName nameIn = return nameIn  -- Just output back the input string value
+handlerName :: String -> Handler String
+handlerName = return  -- Just output back the input string value
```

Comment:
As this code is intended for beginners, I think it might be slightly confusing for them if we include this change. 

Can you please revert it here and in other places so that we keep it as simple as possible? But I think we can replace `return` with `pure`, if you would like to include it.

Reply:
I'm going to revert that thank you. I wasn't thinking about this bing confusing to beginners, and it certainly might be.

I will just change it back to return as replacing it with pure when not doing so in other similar places might as well confuse beginners. Due to me not being super experienced in Haskell I wouldn't trust myself finding all the places that would need to be changed.